desktop/renderer/index.js

```diff
@@ -77,16 +79,31 @@ function setupApp (store) {
   })
 
   const currentWindow = electron.remote.getCurrentWindow()
-  currentWindow.on('focus', () => {
-    store.dispatch(changedFocus(true))
-  })
-  currentWindow.on('blur', () => {
-    store.dispatch(changedFocus(false))
-  })
+  // This fixes reload problems with stale listeners
```

Comment:
I'm a bit puzzled by this. What would cause a stale listener? Is this code running on each HMR?